docs/eventModelling.md

```diff
@@ -0,0 +1,48 @@
+#Event Modelling
+In order for the benefits of event normalisation to be realised, it is necessary for similar events to 
+be represented consistently within the schema.
+
+The following table provides a number of examples of how some common event types can be modelled.
+
+Event Description|Schema Action within ```<EventDetail>``` and Additional Elements | Notes
+---|---|---
+User fails to log onto a workstation due to incorrect password.|```<Authenticate>``` with ```<Outcome>``` subelements ```<Success>``` set to false and ```<Reason>``` set to an appropriate value, e.g. ```IncorrectUsernameOrPassword```.
+Network file server reports that a user has authenticated by presenting kerberos token.|```<Authenticate>```
+Network file server checks that the authenticated user has permission to access a specific folder|This is considered internal system behaviour; nothing has changed as a result of the action. Therefore this is not modelled within the schema.|If there is a requirement to record these events, then ```<Unknown>``` should be used.
+Network file server reports that a user has failed to access a file within a specific folder due to insufficient privilege.|```<View>``` of type ```<File>``` with ```<Permitted>``` element of ```<Outcome>``` set to false, and ```<Description>``` element assigned to a suitable string literal. 
+User writes an email and preses the send button.|```<Send>``` of type ```<Email>```.|
```

Comment:
I think `Create>` and `<Email>` has been used but I am not mad keen on that, unless you are creating a draft